### PR TITLE
[ES] Service.md

### PR DESCRIPTION
**Pull request by MartinEliasQ**
### Describe your changes

- Added service.md file with
- Change Linked Concept in Followiing files: 
    - content/es/abstraction.md
    - content/es/blue-green-deployment.md
    - content/es/event-driven-architecture.md
    - content/es/function-as-a-service.md
    - content/es/microservices-architecture.md
    - content/es/service-mesh.md

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/service.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Servicio
+status: Completed
+category: concept
```
Comment (electrocucaracha) [inline content/es/service.md:4]:
```suggestion
category: Concepto
```

File: content/es/microservices-architecture.md [hunk 2]
```diff
@@ -21,14 +21,13 @@ La demanda de una funcionalidad concreta no aumentará ni disminuirá necesariam
 Volviendo a nuestro ejemplo de Netflix.
 Digamos que después de una gran campaña de marketing, Netflix observa un gran volumen de inscripciones, pero la descarga de videos se ha mantenido más o menos estable en las primeras horas del día.
 El aumento de suscripciones exige más capacidad de suscripción.
-Tradicionalmente (enfoque monolítico), toda la aplicación tendría que ser [escalada](/es/scalability/) para acomodar el aumento - un uso muy ineficiente de los recursos. 
+Tradicionalmente (enfoque monolítico), toda la aplicación tendría que ser [escalada](/es/scalability/) para acomodar el aumento - un uso muy ineficiente de los recursos.
 
 Las arquitecturas monolíticas también hacen que los desarrolladores tengan la posibilidad de caer en errores del diseño.
 Como todo el código está en el mismo sitio, es más fácil hacer que ese código esté [estrechamente acoplado](/tightly-coupled-architectures/) y más difícil aplicar el principio de separación de preocupaciones.
 Los monolitos también suelen requerir que los desarrolladores entiendan todo el código base antes de desplegar cualquier funcionalidad.
 La arquitectura de microservicios es una respuesta a estos retos.
```
Comment (electrocucaracha) [inline content/es/microservices-architecture.md:30]: Este espacio adicional viene de la versión en inglés
```suggestion

```

File: content/es/service.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+tags: ["application", "fundamental", ""]
+---
+
+Es importante señalar que en TI, el término 'servicio' tiene diversos significados. En esta definición, nos enfocaremos en el más tradicional: servicio en términos de microservicio.
```
Comment (electrocucaracha) [inline content/es/service.md:8]:
```suggestion
Es importante señalar que en TI, el término 'servicio' tiene diversos significados.
En esta definición, nos enfocaremos en el más tradicional: servicio en términos de microservicio.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/microservices-architecture.md [hunk 4]
```diff
@@ -21,14 +21,12 @@ La demanda de una funcionalidad concreta no aumentará ni disminuirá necesariam
...
```
[the remark this answers is not in the document]
Comment (MartinEliasQ) [reply content/es/microservices-architecture.md:29]: Ah vale, gracias @electrocucaracha

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Review by ramrodo — APPROVED** [review]
LGTM

**Pull request merged**